### ISSUE TITLE
[Enhancement] Auto-suggest text for tags as well

### ISSUE THREAD

**Issue by the-solipsist** [hpdeifel/hledger-iadd#64]
Many times for similar transactions, similar tags need to be applied as well.  So it would be helpful to auto-suggest tags on the basis of previously used tags.

**Comment by hpdeifel** [comment]
I agree that this would be useful. `iadd` currently has no knowledge of tags. We could suggest the comment from the old transaction in the comment input field, but the input box for the comment is currently an oddball in the user interface and doesn't support suggestions.

Maybe it would also be time to add dedicated support for tags. If you have ideas on how such a feature could look, I would be happy to hear them.
